src/libraries/Common/tests/System/Net/Prerequisites/NetCoreServer/GenericHandler.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace NetCoreServer
```

Comment:
I'm not sure if this is best name since .NET Core become just .NET in 5.0. 
Since this is purely related to testing perhaps the name can reflect that.